packages/editor-ui/src/components/Sticky.vue

```diff
@@ -139,9 +148,36 @@ export default defineComponent({
 			type: Number,
 			default: GRID_SIZE,
 		},
+		name: {
+			type: String,
+			required: true,
+		},
+		instance: {
+			type: Object as PropType<BrowserJsPlumbInstance>,
+			required: true,
+		},
+		isReadOnly: {
+			type: Boolean,
+		},
+		isActive: {
+			type: Boolean,
+		},
+		hideActions: {
+			type: Boolean,
+		},
+		disableSelecting: {
+			type: Boolean,
+		},
+		showCustomTooltip: {
+			type: Boolean,
+		},
+		workflow: {
+			type: Object as () => Workflow,
+			required: true,
+		},
```

Comment:
is there a plan for how we want to tackle this props duplication once we have migrated away from `defineComponent` ?